Demos/01-create-app/README.md

```diff
@@ -1,3 +1,5 @@
 # Completed module: Create an ASP.NET MVC web app
 
-The version of the project in this directory reflects completing the tutorial up through [Create an ASP.NET MVC web app](https://docs.microsoft.com/graph/training/aspnet-tutorial?tutorial-step=1). If you use this version of the project, you need to complete the rest of the tutorial starting at [Register the app in the portal](https://docs.microsoft.com/graph/training/aspnet-tutorial?tutorial-step=2).
\ No newline at end of file
+The version of the project in this directory reflects completing the tutorial up through [Create an ASP.NET MVC web app](https://docs.microsoft.com/graph/training/aspnet-tutorial?tutorial-step=1). If you use this version of the project, you need to complete the rest of the tutorial starting at [Register the app in the portal](https://docs.microsoft.com/graph/training/aspnet-tutorial?tutorial-step=2).
+
+Installing NuGet packages adds content to the /packages, /Content & /Scripts folders. The files within the /Content & /Scripts folders frequently contain 3rd party libraries which do not include redistributable licenses. Therefore the files in these folders are not included in the repository. The NuGet restore process will not repopulate these folders, only a NuGet reinstall of the package will do that. Therefore you will need to reinstall all packages using the NuGet CLI to repopulate these folders if you want to run the final built solution locally. Refer to the following for more information: [Microsoft Docs – How to reinstall and update packages](https://docs.microsoft.com/en-us/nuget/consume-packages/reinstalling-and-updating-packages).”
```

Comment:
Removing the Content and Scripts folders breaks building these project unless you do the manual steps in PMC to reinstall everything. This is bad from a DevX standpoint. The files in those folder are all distributed under the MIT license which should be fine. Please put these back.

Reply:
I agree... will put these back, however you should be aware of the following.

We are under instructions by OfficeDev to not distribute 3rd party code. In the past security / quality scans of the OfficeDev/TrainingContent repo have identified code comments with swear words & other content MSFT deemed unacceptable. We have to get specific libraries approved by LCA and so far that's only included jQuery & Bootstrap. This was the compromise.